easybuild/easyconfigs/s/Salmon/Salmon-0.11.2-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '0.11.2'
+versionsuffix = '-Python-2.7.14'
```

Comment:
@HPC-UniOldenburg This versionsuffix doesn't make much sense, since `Python` is not a dependency at all (not even indirectly)?